### PR TITLE
README should mention that source download refers to MDK and link to the better documentation #2260

### PR DESCRIPTION
**Pull request by terranix**
I also changed this to a readme. Check out what it looks like on my fork.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Installation Instructions
+
+## Mod Users
+
+Download the latest **Installer** (or **Installer-win** for Windows users) from http://files.minecraftforge.net and follow the installation instructions.
+
+## Mod Devs
+
+Read the [MinecraftForge Documentation]([MinecraftForge Documentation](http://http://mcforge.readthedocs.io/)
```
Comment (diesieben07) [inline README.md]: Ahem.

**Comment by diesieben07** [comment]
While you are at it... this whole crap about gradle could be cleaned up. There is no need to separate between windows and unix-like, as windows appends a `.bat` automatically, so you can have `./gradlew task` for both. 
Also I don't think we need to mention the different ways of invoking gradle (wrapper vs. installation) this often. Once is enough.

**Comment by terranix** [comment]
I was thinking the same thing, but I was wondering if that was too big a change. I'm going to do it though I have a version almost done

File: README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    git clone git@github.com:MinecraftForge/MinecraftForge.git
+    gradle setupForge
+
+To use Eclipse, point your Eclipse workspace at the `eclipse` folder.
+
+## Development Requirements 
+(For mod devs and contributors)
+
+  - JDK
+  - [Gradle](https://docs.gradle.org/current/userguide/installation.html) is recommended, but you can use  `gradlew` (or `gradlew.bat`) instead.
```
Comment (diesieben07) [inline README.md]: Actually Gradle recommends using the wrapper instead of the installation.

**Comment by terranix** [comment]
Then we should just document that by default and note that Gradle can also be used.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Installation Instructions
+
+## Mod Users
+
+Download the latest **Installer** (or **Installer-win** for Windows users) from http://files.minecraftforge.net and follow the installation instructions.
+
+## Mod Devs
+
+Read the [MinecraftForge Documentation](http://http://mcforge.readthedocs.io/)
```
Comment (jamierocks) [inline README.md]: http://http://

File: README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    
+You can install the [Gradle Plugin for Eclipse](https://marketplace.eclipse.org/content/gradle-sts-integration-eclipse) and import the MDK folder.
+
+To get the decompiled classes:
+
+    gradlew setupDecompWorkspace
+
+##  Contributors
+
+    git clone git@github.com:MinecraftForge/MinecraftForge.git
```
Comment (jamierocks) [inline README.md]: Probably a better idea to use https://

File: README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+    gradlew eclipse 
+    
+You can install the [Gradle Plugin for Eclipse](https://marketplace.eclipse.org/content/gradle-sts-integration-eclipse) and import the MDK folder.
+
+To get the decompiled classes:
+
+    gradlew setupDecompWorkspace
+
+##  Contributors
```
Comment (jamierocks) [inline README.md]: Two spaces

**Comment by Actuarius** [comment]
@mezz added labels [Cleanup]

**Pull request closed without merging**